doc/nrf/ug_matter_configuring_controller.rst

```diff
@@ -54,7 +54,7 @@ CHIP Tool for Android
 *********************
 
 CHIP Tool for Android (also known as Android CHIPTool) is the recommended Mobile controller for mobile, which allows you to test Matter applications using an Android smartphone.
-You can use it in the :ref:`development environment setup option <ug_matter_configuring>` where Thread Border Router and Matter controller are running on separate devices.
+You can use it in the :ref:`development environment setup option <ug_matter_configuring>` where the IPv6 network device and Matter controller are running on separate devices.
```

Comment:
This is similar comment, as above. In setup with Thread we've got additional device that is Thread Border Router, but for Wi-Fi there isn't corresponding device for it, so I'm not sure what is IPv6 network device here.